services/ui-src/src/page/section/DetailSection.tsx

```diff
@@ -121,7 +122,9 @@ export const DetailSection = ({
                                   Workflow.PACKAGE_ACTION.RESPOND_TO_RAI
                                 ? () => {
                                     onLinkAction({
-                                      href: pageConfig.raiLink,
+                                      href: TYPE_TO_RAI_ROUTE[
+                                        detail.componentType
+                                      ],
```

Comment:
I decided to remove the rai link per config file because it would duplicate the definition of what the RAI link is from routes.js `TYPE_TO_RAI_LINK` which is required on the package list page. 
We can discuss if anyone thinks this was not a prudent thing to do.

Reply:
Sounds like you made an improvement to me :)